src/animation/Clip.ts

```diff
@@ -102,6 +102,44 @@ class Clip{
         return clip;
     }
 
+    static fromThree( anim: any, arm: any ) : Clip {
+        // Convert from THREE.AnimationClip to OSSOS.Clip (needs armature for bone names)
+        const clip = new Clip();
+        clip.name = anim.name;
+        clip.duration = anim.duration;
+        clip.frameCount = anim.tracks[0].times.length;
+        clip.timeStamps = [anim.tracks[0].times];   // Limit: Only one timestamp per clip
+        
+        // Add tracks
+        clip.tracks = [];
+        let track    : ITrack;          // Animator Track    
+        for ( const t of anim.tracks ) {
+                
+            switch ( t.constructor.name ) {
```

Comment:
I encountered issues with the minified bundle when using `constructor.name`, as the bundler sometimes changes object names, leading to crashes. A possible solution is to adjust the bundler settings and use Terser. However, if there's a way to avoid relying on constructor.name, it might prevent this issue. Just wanted to share in case it's helpful.

Reply:
The safest solution I think might be to use instanceof conditions. There is only two Track types in ossos but more in 3js. Something it should be easy to just do

if( t instanceof VectorKeyframeTrack )                track = new Vec3Track();
else if( t instanceof QuaternionKeyframeTrack ) track = new QuatTrack();
else continue;

I've rebuilt the animation system for the next version of ossos, So there are big changes in alot of places.
https://github.com/sketchpunklabs/ossos/tree/ossos_next